docs/scanners.md

```diff
@@ -171,3 +172,31 @@ class MyKeyboard(KMKKeyboard):
         # etc...
     ]
 ```
+#### Multiple Scanners coord_mapping and keymap changes
+To add more scanners you need to add onto your your " coord_mapping " Example below.
+
+Before:
```

Comment:
instead of "before/after" use something more descriptive, like "here's a `coord_mapping` for a split with single 30-key scanner", "and here's the same split with two scanners, the 30-key matrix and an encoder".